pkg/cmd/clientConfig_test.go

```diff
@@ -141,7 +142,8 @@ func TestClientConfigCmd_GetClientConfigCmd(t *testing.T) {
 							ObjectMeta: metav1.ObjectMeta{
 								Name: "keycloak",
 								Labels: map[string]string{
-									"mobile": "enabled",
+									"mobile":         "enabled",
+									"mobileClientId": "client-id",
```

Comment:
this label should  be `clientId` the APBs will be updated to use this string as the key.